tornjak-frontend/src/components/types.ts

```diff
@@ -0,0 +1,99 @@
+// agents
+
+export interface SPIFFEIDType {
+  // From https://github.com/spiffe/spire-api-sdk/blob/main/proto/spire/api/types/spiffeid.pb.go
+  // Trust domain portion the SPIFFE ID (e.g. "example.org")
+  trust_domain: string;
+  // The path component of the SPIFFE ID (e.g. "/foo/bar/baz"). The path
+	// SHOULD have a leading slash. Consumers MUST normalize the path before
+	// making any sort of comparison between IDs.
+  path: string;
+}
+
+export interface SelectorType {
+  // From https://github.com/spiffe/spire-api-sdk/blob/main/proto/spire/api/types/selector.pb.go
+  // The type of the selector. This is typically the name of the plugin that
+	// produces the selector.
+  type: string;
+  value: string;
+}
+
+export interface AgentsListType {
+  // From https://github.com/spiffe/spire-api-sdk/blob/main/proto/spire/api/types/agent.pb.go
+  id: SPIFFEIDType; // SPIFFE ID of the agent.
+  attetstation_type: string; // The method by which the agent attested.
+  x509svid_serial_number: string; // The X509-SVID serial number.
+  x509svid_expires_at: number; //  The X509-SVID expiration (seconds since Unix epoch).
```

Comment:
to avoid casting issues, is there a better matching type to map here?

Reply:
Not sure I understand, can you elaborate?

Reply:
resolved to bigint